back/src/Guide/Matomo.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+{- | Functions for interacting with Matomo (<https://matomo.org/>, our web analytics). -}
```

Comment:
Oh, and can you also add a link to Matomo docs?

https://developer.matomo.org/api-reference/tracking-api